src/components/application_manager/include/application_manager/smart_object_keys.h

```diff
@@ -216,6 +216,7 @@ extern const char* key_press_mode;
 // duplicate names from hmi_request
 extern const char* limited_character_list;
 extern const char* auto_complete_text;
+extern const char* auto_complete_list;
```

Comment:
@dcherniev You can use whether, first or second. I don't remember any agreements about these namespaces.